bpaingest/projects/stemcells/ingest.py

```diff
@@ -835,6 +864,7 @@ class StemcellsMetabolomicsAnalysedMetadata(BaseMetadata):
             fld('data_type', 'data type'),
             fld('analysis_file_name', 'file name of analysed data (folder or zip file) file name'),
             fld('additional_comments', 'additional comments'),
+            skp(re.compile(r'^x{4,6}$'), skip_all=True)
         ],
```

Comment:
this seems really broad, are you sure this is correct? could the re be more specific?